src/Form.php

```diff
@@ -167,14 +168,11 @@ protected function initLayout(): void
                 ->addMoreInfo('layout', $this->layout);
         }
 
-        // allow to submit by pressing an enter key when child control is focused
-        $jsSubmit = $this->js(false, null, $this->formElement)->form('submit');
-        $this->on('submit', new JsExpression('if (event.target === this) { []; }', [$jsSubmit]));
-
-        // Add save button in layout
+        // add save button in layout
         if ($this->buttonSave) {
             $this->buttonSave = $this->layout->addButton($this->buttonSave);
             $this->buttonSave->setAttr('tabindex', 0);
+            $jsSubmit = $this->js()->form('submit');
```

Comment:
Invoke `div.ui.form` `submit`, thanks to l138, the submit will be sent to the native form tag.